src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -31,7 +31,6 @@ def __init__(self, *args, **kwargs):
         self.stdout = logging.getLogger('avocado.test.stdout')
         self.progress_log = logging.getLogger("progress")
         self.module_init = False
-        self.provider = None
```

Comment:
Even though `get_env()` further down has `get_env`, it's better to keep this here, since `__init__` is the ideal place to initialize variables. And looking in `__init__` gives an overview of what all could be set

Reply:
Not quite sure what you mean here. this variable is completely gone now as not needed

Reply:
Woops. I was looking at the diff of just the last commit. Sorry :)